app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/AddPictureActivityTest.kt

```diff
@@ -41,22 +31,6 @@ class AddPictureActivityTest {
         PermissionGranter.allowPermissionOneTime(Manifest.permission.CAMERA)
     }
 
-    // inspired by : https://stackoverflow.com/a/35924943/7158887
-    private fun waitFor(millis: Long): ViewAction =
-        object : ViewAction {
-            override fun getConstraints(): Matcher<View> {
-                return Matchers.anyOf(isRoot(), withId(R.id.cameraImageView))
-            }
-
-            override fun getDescription(): String {
-                return "Wait for $millis milliseconds"
-            }
-
-            override fun perform(uiController: UiController?, view: View?) {
-                uiController?.loopMainThreadForAtLeast(millis)
-            }
-        }
-
```

Comment:
Maybe we put this function under Test/utils for example for refactoring purposes as most tests currently are using it

Reply:
That's what I did x) The left column is how the code was before modification